apps/reference/docs/guides/platform/disk-usage.mdx

```diff
@@ -14,6 +14,33 @@ SELECT SUM(pg_database_size(pg_database.datname)) / (1024 * 1024) as db_size_mb
 
 This value is also reported in the [database settings page](https://app.supabase.com/project/_/settings/database).
 
+## Full disk prevention
+
+Running out of disk space will cause Postgres to become unstable, and operations such as backups and automatic service optimization to be disrupted.
+
+In order to prevent this your project will go into read-only mode once it reaches 95% disk usage, preventing writes to the database.
+In read-only mode clients will start encountering errors such as `cannot execute INSERT in a read-only transaction`.
+
+Regular operation (read-write mode) will be automatically enabled once disk usage falls under 95%.
```

Comment:
```suggestion
Running out of disk space causes operations such as backups and automatic service optimization to be disrupted.
To prevent this, your project goes into read-only mode once it reaches 95% disk usage, preventing writes to the database. In read-only mode, clients will encounter errors such as `cannot execute INSERT in a read-only transaction`.

Regular operation (read-write mode) is automatically enabled once disk usage falls under 95%.
```

Reply:
That's great! Wonder if leaning a bit more on the chance that the database service might be impacted could help inform better. 

Maybe something like the following?
```
Running out of disk space causes the database service to become unresponsive. Operations such as backups and automatic service optimization are also disrupted.
```

Reply:
👍🏼 looks good!